lms/djangoapps/labster_course_license/utils.py

```diff
@@ -105,8 +106,13 @@ def course_tree_info(store, simulations, licensed_simulations):
     Retuns information about the course's xblocks.
     """
     course_info = {}
+    invalid_simulations = []
+    validation_pattern = r'^https:\/\/[\w\d]*\.labster\.com\/[a-z]*\/[\w\d]*\/$'
```

Comment:
> It makes sense 'cause we can have tabs/spaces/other chars before/after url

but this particular functionality works only with `simulation_id`. And the problems were with them. 
I believe that LTI module cut all whitespaces at the beginnig and at the end before save, but you have to double check that.
Also, I don't like the idea of hardcoding protocol and labster.com here. Probably we can check for white spaces, tabulations, etc..